openlibrary/components/MergeUI/AuthorRoleTable.vue

```diff
@@ -19,6 +19,9 @@
                                 {{role[field].key.slice("/authors/".length)}}
                             </a>
                         </div>
+                        <div v-else-if="field == 'name'">
+                            {{role[field]}}
+                        </div>
                         <div v-else>{{a[k]}}</div>
```

Comment:
Ahhh I see what you mean; it's because there's a bug in the original code! 😁 `a` and `k` are undefined. This should do the trick:

```suggestion
                        <div v-else>{{role[field]}}</div>
```